utils/python/CIME/utils.py

```diff
@@ -21,8 +21,9 @@ def expect(condition, error_msg, exc_type=SystemExit):
     """
     if (not condition):
         # Uncomment these to bring up a debugger when an expect fails
-        #import pdb
-        #pdb.set_trace()
+        if logger.isEnabledFor(logging.DEBUG):
```

Comment:
Yeah, I also uncomment these lines frequently. I think this is a reasonable change; it might be better to add another standard option to control this, but I think this is fine for now. You may want to remove the "Uncomment these ..." comment above.